frontend/src/components/maps/leaflet/SvgMarkers/DraftMarker.tsx

```diff
@@ -74,6 +74,7 @@ export const DraftMarker: React.FunctionComponent<IDraftMarkerProps> = ({ text }
         <text x={text?.length === 1 ? 7 : 4} y="15" textLength="1.25rem">
           {text?.length ?? 0 <= 2 ? text : '..'}
         </text>
+        {children}
```

Comment:
this allows the circle icon to be drawn within this draft marker - instead of having two duplicated draft marker svg components, one with the circle icon and one without.